### PR TITLE
picgo: Add version 2.2.2

### PR DESCRIPTION
**Pull request by huangnauh**

### CODE REVIEW

File: bucket/picgo.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "64bit": {
+            "url": "https://github.com/Molunerfinn/PicGo/releases/download/v2.3.0-beta.4/PicGo-Setup-2.3.0-beta.4.exe#/dl.7z",
+            "hash": "86057615503ee7a62fb1c837b7db7856a19ec5e9efa18b93a3b3b71c25af9fd4",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
+            ]
+        }
+    },
+    "shortcuts": [["PicGo.exe", "PicGo"]],
```
Comment (Ash258) [inline bucket/picgo.json:16]: format with bin/formatjson.ps1

File: bucket/picgo.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.3.0-beta.4",
+    "homepage": "https://picgo.github.io/PicGo-Doc/en/",
+    "description": "🚀A simple & beautiful tool for pictures uploading built by vue-cli-electron-builder",
```
Comment (Ash258) [inline bucket/picgo.json:4]: Having emojis in decription is completely absurd.

File: bucket/picgo.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "checkver": {
+        "url": "https://github.com/Molunerfinn/PicGo/releases/latest",
+        "regex": "/PicGo-Setup-([\\w.-]+).exe"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Molunerfinn/PicGo/releases/download/v$version/PicGo-Setup-$version.exe#/dl.7z"
+            }
+        }
```
Comment (Ash258) [inline bucket/picgo.json:26]: Add hash extraction from $baseurl/latest.yml

File: bucket/picgo.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
+            ]
+        }
+    },
+    "shortcuts": [["PicGo.exe", "PicGo"]],
+    "checkver": {
+        "url": "https://github.com/Molunerfinn/PicGo/releases/latest",
+        "regex": "/PicGo-Setup-([\\w.-]+).exe"
```
Comment (Ash258) [inline bucket/picgo.json:19]: Do not use beta releases

File: bucket/picgo.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.3.0-beta.4",
```
Comment (Ash258) [inline bucket/picgo.json:2]: Update to 2.2.2

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by huangnauh** [comment]
@Ash258 please take a look

File: bucket/picgo.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "shortcuts": [
+        [
+            "PicGo.exe",
+            "PicGo"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/Molunerfinn/PicGo/releases/latest",
+        "regex": "/PicGo-Setup-([\\d.]+).exe"
```
Comment (Ash258) [inline bucket/picgo.json:24]:
```suggestion
        "url": "https://github.com/Molunerfinn/PicGo/releases",
        "regex": "/PicGo-Setup-([\\d.]+)\\.exe"
```

File: bucket/picgo.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+        "64bit": {
+            "url": "https://github.com/Molunerfinn/PicGo/releases/download/v2.2.2/PicGo-Setup-2.2.2.exe#/dl.7z",
+            "hash": "sha512:18144b18773c4ddb5aaebd3b51bc1e8c437deed80e0bdf1029d033d7374d56cc001052a4225a8901d3f85dd9408abf394cb7745de0164623240913cc7b9623d9",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
+            ]
+        }
+    },
+    "shortcuts": [
```
Comment (Ash258) [inline bucket/picgo.json:16]: https://github.com/Molunerfinn/PicGo/blob/dev/src/main/utils/handleArgv.ts

```suggestion
    "bin": "PicGo.exe",
    "shortcuts": [
```

File: bucket/picgo.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.2.2",
+    "homepage": "https://picgo.github.io/PicGo-Doc/en/",
+    "description": "A simple & beautiful tool for pictures uploading built by vue-cli-electron-builder",
```
Comment (Ash258) [inline bucket/picgo.json:4]:
```suggestion
    "description": "Image uploader/manager",
    "homepage": "https://picgo.github.io/PicGo-Doc/en/",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Next time you will not follow github flow with accepting suggestion via UI all your PRs will be closed.

**Comment by huangnauh** [comment]
> Next time you will not follow github flow with accepting suggestion via UI all your PRs will be closed.

Sorry for the late reply, I received the github suggestion after you reminded me. 
Before that, I only received comments, so I committed the fix via git command instead of ui.

**Pull request merged**